File: legacy/legacy-apis/Psi.Models.ClientConfigurationModels/Admin/AccountNumberReassignmentSettings.cs
namespace Psi.Data.Models.ClientConfigurationModels.Admin
{
    public class AccountNumberReassignmentSettings : SettingsBaseHelper
    {
        public AccountNumberReassignmentSettings(ISettingsBase settingsBase) : base(settingsBase)
        {
        }

        /// <summary>
        /// If the Uuid should be changed during an account number reassignment
        /// </summary>
        [SettingKey("Admin.Account.AccountNumberReassignment.ChangeUuidWhenAccountNumberMatches")]
        public bool ChangeUuidWhenAccountNumberMatches
        {
            get => GetBoolValue();
            set => SetValue(value);
        }

        /// <summary>
        /// If Mfa questions should be removed upon account number reassignment
        /// </summary>
        [SettingKey("Admin.Account.AccountNumberReassignment.RemoveMfaQuestion")]
        public bool RemoveMfaQuestion
        {
            get => GetBoolValue();
            set => SetValue(value);
        }

        /// <summary>
        /// If scheduled transfers using account number should be disabled
        /// </summary>
        [SettingKey("Admin.Account.AccountNumberReassignment.DisableScheduledTransfersUsingAccount")]
        public bool DisableScheduledTransfersUsingAccount
        {
            get => GetBoolValue();
            set => SetValue(value);
        }
    }
}

File: workspace/legacy-analyzer/output/infrastructure/models/Admin/AccountNumberReassignmentSettings.ts
// Generated imports

export interface AccountNumberReassignmentSettings {
    /** @settingKey Admin.Account.AccountNumberReassignment.ChangeUuidWhenAccountNumberMatches */
    /**
     * /// <summary>
     * /// /// <summary>
     * /// /// If the Uuid should be changed during an account number reassignment
     * /// /// </summary>
     * /// </summary>
     */
    changeUuidWhenAccountNumberMatches: boolean;
    /** @settingKey Admin.Account.AccountNumberReassignment.RemoveMfaQuestion */
    /**
     * /// <summary>
     * /// /// <summary>
     * /// /// If Mfa questions should be removed upon account number reassignment
     * /// /// </summary>
     * /// </summary>
     */
    removeMfaQuestion: boolean;
    /** @settingKey Admin.Account.AccountNumberReassignment.DisableScheduledTransfersUsingAccount */
    /**
     * /// <summary>
     * /// /// <summary>
     * /// /// If scheduled transfers using account number should be disabled
     * /// /// </summary>
     * /// </summary>
     */
    disableScheduledTransfersUsingAccount: boolean;
}
